### PR TITLE
update kata-deploy readme's ToC

### PR DESCRIPTION
**Pull request by egernst**

### CODE REVIEW

**Comment by egernst** [comment]
applicable to 1.5 stable branch

File: kata-deploy/README.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
+    + [Run a sample workload](#run-a-sample-workload)
+    + [Remove Kata from the Kubernetes cluster](#remove-kata-from-the-kubernetes-cluster)
   * [kata-deploy details](#kata-deploy-details)
     + [Dockerfile](#dockerfile)
-    + [Daemonsets and RBAC](#daemonsets-and-rbac-)
-      - [runtime-labeler](#runtime-labeler-)
-      - [CRI-O and containerd kata installer](#cri-o-and-containerd-kata-installer-)
-    + [Kata cleanup](#kata-cleanup-)
-
+    + [Daemonsets and RBAC:](#daemonsets-and-rbac-)
```
Comment (chavafg) [inline kata-deploy/README.md]: remove last dash?

File: kata-deploy/README.md [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
+      - [Kata installer: kata-deploy](#kata-installer--kata-deploy)
```
Comment (chavafg) [inline kata-deploy/README.md]: `#kata-installer-kata-deploy`

File: kata-deploy/README.md [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
+    + [Kata cleanup:](#kata-cleanup-)
```
Comment (chavafg) [inline kata-deploy/README.md]: `[Kata cleanup](#kata-cleanup)` ?

Comment (egernst) [reply kata-deploy/README.md]: The `:` was being resolved to `-` in the TOC.  I removed the `:`s :)

Comment (jodh-intel) [reply kata-deploy/README.md]: https://github.com/isaacs/github/issues/215 stings.

/me tries valiantly not to preach the benefits of reStructuredText which auto-generates TOCs... even on github... Wait... :smile:

**Pull request merged**